app/javascript/dashboard/components/widgets/conversation/WhatsappTemplates/TemplateParser.vue

```diff
@@ -43,8 +41,6 @@
 </template>
 
 <script>
-import { required } from 'vuelidate/lib/validators';
```

Comment:
`requiredIf` is better instead of removing the required condition altogether, right?